server/services/bookerService.ts

```diff
@@ -1,10 +1,16 @@
 import logger from '../../logger'
 import { HmppsAuthClient, OrchestrationApiClient, RestClientBuilder } from '../data'
-import { AuthDetailDto, PrisonerInfoDto, VisitorInfoDto } from '../data/orchestrationApiTypes'
+import { AuthDetailDto, VisitorInfoDto } from '../data/orchestrationApiTypes'
 import { isAdult } from '../utils/utils'
 
-export interface Prisoner extends PrisonerInfoDto {
+export type Prisoner = {
```

Comment:
Defining `Prisoner` here rather than extending the type from the back-end. This is to give a flat structure and only the properties that are used.